autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/daos/configurations/ConfigureDatabaseDao.java

```diff
@@ -82,97 +87,88 @@ protected void initDao() throws Exception {
     }
 
     /**
-     * TODO Documentation
-     *
-     * @return
+     * It returns true if has the 'administration_status' column at the 'host' table.
      */
-    private boolean hasHostConsolidationStatusColumn() {
+    protected boolean hasHostConsolidationStatusColumn() {
         return !getJdbcTemplate().queryForList(sqlHasHostAdministrationStatusColumn).isEmpty();
     }
 
     /**
-     * TODO Documentation
+     * It creates the 'administration_status' column at the 'host' table..
      */
-    private void createHostConsolidationStatusColumn() {
+    protected void createHostConsolidationStatusColumn() {
         getJdbcTemplate().execute(sqlCreateHostAdministrationStatusColumn);
     }
 
     /**
-     * TODO Documentation
-     *
-     * @return
+     * Returns true if has the 'start_type' column at the 'host' table.
      */
-    private boolean hasHostStartTypeColumn() {
+    protected boolean hasHostStartTypeColumn() {
         return !getJdbcTemplate().queryForList(sqlHasHostStartTypeColumn).isEmpty();
     }
 
     /**
-     * TODO Documentation
+     * It creates the 'start_type' column at the 'host' table.
      */
-    private void createHostStartTypeColumn() {
+    protected void createHostStartTypeColumn() {
         getJdbcTemplate().execute(sqlCreateHostStartTypeColumn);
     }
 
     /**
-     * TODO Documentation
-     *
-     * @return
+     * It returns true if has the 'administration_status' column at the 'cluster' table.

```

Comment:
It returns true there is a 'administration_status' column at the 'cluster' table.